data/meson.build

```diff
@@ -59,12 +59,15 @@ if build_daemon
     install_dir: join_paths(datadir, 'dbus-1', 'system.d')
   )
 
+  build_conf = configuration_data()
+  build_conf.set('libexecdir', libexecdir)
+
   if libsystemd.found()
-    con2 = configuration_data()
-    con2.set('libexecdir', libexecdir)
-    con2.set('bindir', bindir)
-    con2.set('datadir', datadir)
-    con2.set('localstatedir', localstatedir)
+    build_conf = configuration_data()
```

Comment:
isn't this going to axe the original declaration on line 62 (and thus lose libexecdir declaration)?  I think this line can go away and just re-use `build_conf` in the scope above